src/config/dark-sites.config

```diff
@@ -1241,25 +1149,20 @@ viddit.red
 villains.fandom.com
 vimm.net
 vinesauce.com
-visualboxsite.com
-visuallysynced.xyz
 vivek9patel.github.io
 vixeny.dev
 vleer.app
 vogons.org
-volksverpetzer.social
 volta.net
 vrv.co
 vscode.dev
 vudu.com
 vuecinemas.nl
 w0rp.com
+w3schools.com
```

Comment:
Site with button to switch dark/light mode:
![obraz](https://github.com/user-attachments/assets/63030540-f6f7-4bb4-8808-67ba87689dc1)

To remove!